File: LeetCode/C#/1. Two Sum.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        var indices = new int[2];
        
        for(var i = 0; i < nums.Length - 1; i++)
        {
            for(var j = i + 1; j < nums.Length; j++)
            {
                if(nums[i] + nums[j] == target)
                {
                    indices[0] = i;
                    indices[1] = j;
                }
            }
        }
        
        return indices;
    }
}

File: LeetCode/C++/1. Two Sum.cpp
class Solution
{
public:
    vector<int> twoSum(vector<int> &nums, int target)
    {
        map<int, int> numMap;

        for (int i = 0; i < nums.size(); i++)
        {
            int complement = target - nums[i];

            if (numMap.count(complement))
            {
                return {i, numMap[complement]};
            }

            numMap.insert({nums[i], i});
        }

        return {};
    }
};